documentation/example_codes/ExampleJobExecutor.m

```diff
@@ -0,0 +1,38 @@
+% Example object which just demonstrates the methods and variables available in JobExecutor
+% and where they come from
+classdef ExampleJobExecutor < JobExecutor
+    methods
+        function obj = ExampleJobExecutor()
+        % Constructor cannot take args as constructed by JobDispatcher
+        end
```

Comment:
I would say, should have empty constructor as will be initialized by predefined init method. 
And write description on how to use properties of existing JE to initialize the particular job.